internal/transaction/response/menu_response.go

```diff
@@ -0,0 +1,18 @@
+package response
+
+import "github.com/jackc/pgx/v5/pgtype"
+
+type CreateMenuResponse struct {
+	ID          pgtype.UUID `gorm:"primaryKey;type:uuid"`
+	Price       float64     `gorm:"column:m_price"`
+	Description string      `gorm:"column:m_description"`
+	Url         string    `gorm:"type:text"`
+}
```

Comment:
Wrong meta data. You shouldn't write gorm here because it's just response struct. It don't interact with database or gorm. You should change it to json (`json:"something"`) for interact with internet